data-protocols/ids/ids-api-multipart-dispatcher-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/sender/MultipartCatalogDescriptionRequestSender.java

```diff
@@ -84,7 +93,13 @@ protected Message buildMessageHeader(CatalogRequest request, DynamicAttributeTok
         message.setProperty(Range.TO, request.getRange().getTo());
         return message;
     }
-
+    
+    /**
+     * Parses the response content and extracts the catalog from the received self description.
```

Comment:
Shouldn't this method return a `MultipartResponse<Catalog>` to align with the other senders?

Reply:
changed

Reply:
will be added in #1695